extensions/wikia/ContributionPrototype/ContributionPrototype.setup.php

```diff
@@ -12,5 +12,16 @@
 	}
 });
 
-// take over actions
+// take over view action
 $wgActions['view'] = ContributionPrototype\CPViewAction::class;
+
+// This hook runs some setup/resource loading for the edit action
+$wgHooks[ 'CustomEditor' ][] = 'ContributionPrototype\ContributionPrototypeHooks::onCustomEditor';
+
+// different hook to load some js vars
+$wgHooks['MakeGlobalVariablesScript'][] = 'ContributionPrototype\ContributionPrototypeHooks::onMakeGlobalVariablesScript';
+
+//This hook just lets us override the PHP edit class during the edit action
+//Not the "right" way to do this but we erase any other AlternateEditPageClass hooks this way because there can be only one
+$wgHooks['AlternateEditPageClass'] = ['ContributionPrototype\ContributionPrototypeHooks::onAlternateEditPageClass'];
+#$wgHooks['AlternateEditPageClass'][] = ['ContributionPrototype\ContributionPrototypeHooks::onAlternateEditPageClass'];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)